packages/core/echo/echo-db/src/queue/queue-service.ts

```diff
@@ -0,0 +1,55 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import type { ObjectId } from '@dxos/echo-schema';
+import type { EdgeHttpClient } from '@dxos/edge-client';
+import type { SpaceId } from '@dxos/keys';
+import type { QueryResult, QueueQuery } from '@dxos/protocols';
+
+/**
+ * Service for managing queues.
+ */
+export interface QueuesService {
+  queryQueue(subspaceTag: string, spaceId: SpaceId, query: QueueQuery): Promise<QueryResult>;
+
+  insertIntoQueue(subspaceTag: string, spaceId: SpaceId, queueId: ObjectId, objects: unknown[]): Promise<void>;
+
+  deleteFromQueue(subspaceTag: string, spaceId: SpaceId, queueId: ObjectId, objectIds: ObjectId[]): Promise<void>;
```

Comment:
it seems odd to me that these apis don't align with the ones on `Queue` which call them